deploy/manifests/dev/us-east-2/tenant/storetheindex/dhfind-ago2/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dhfind
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dhfind-ago2
+  template:
+    metadata:
+      labels:
+        app: dhfind-ago2
+    spec:
+      topologySpreadConstraints:
+        - maxSkew: 1
+          topologyKey: topology.kubernetes.io/zone
+          whenUnsatisfiable: ScheduleAnyway
+      containers:
+        - name: dhfind
+          args:
+            - '--dhstoreAddr=http://dhstore-ago2.internal.dev.cid.contact/'
+            - '--stiAddr=http://ago-indexer:3000/'
```

Comment:
Would it be better for this to go to an indexer or to indexstar?

Reply:
this is used for fetching providers info right? So pointing that to indexstar would result in circular query across instances?

Reply:
The `--stiAddr` is only used for fetching provider info. So, I think it makes most sense to point this at the indexer instance that is associated with this dhfind instance.

Reply:
Btw, this can be pointed at `indexstar`. There won't be a circular dependency as the `stiAddr`  endpoint is used for provider lookups only. Pointing that directly at the indexer instance saves up some latency on provider lookups at the cost of not being able to do lookups at all when this indexer instance is down. However such performance boost is negligible because provider records are cached.

So the bottom line is - I recommend pointing `stiAddr` to `indexstar` next time.